src/cmd/sample.rs

```diff
@@ -108,8 +111,19 @@ fn sample_reservoir<R: io::Read>(
         reservoir.push(row?);
     }
 
+    // Seeding rng
+    let mut rng: StdRng = match seed {
+        Some(s) => {
+            let s: &[_] = &[s];
+            SeedableRng::from_seed(s)
+        }
+        None => {
+            let s = ::rand::thread_rng().gen_iter::<usize>().take(256).collect::<Vec<usize>>();
```

Comment:
Could you please follow prevailing style conventions and wrap this code to 79 columns, inclusive? e.g.,

```rust
let s = ::rand::thread_rng()
    .gen_iter()
    .take(256)
    .collect::<Vec<usize>>();
```

I also removed the type annotation from `gen_iter`, but you may need to add it back.

Reply:
Done.